app/views/no_vat_repayments.scala.html

```diff
@@ -16,17 +16,32 @@
 
 @import config.ViewConfig
 @import uk.gov.hmrc.play.views.html._
+@import model.des.BankDetails
 
 @this(
 mainTemplate: views.html.main_template
 )
 
-@()(implicit request: Request[_], messages: Messages, appConfig: ViewConfig)
+@(showBankDetails: Boolean, bankDetails: Option[BankDetails])(implicit request: Request[_], messages: Messages, appConfig: ViewConfig)
```

Comment:
`appConfig: ViewConfig` should be injected.

Reply:
Done